src/SprykerEco/Zed/Unzer/Persistence/Propel/Schema/spy_unzer.schema.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\Unzer\Persistence" package="src.Orm.Zed.Unzer.Persistence">
```

Comment:
Property _defaultIdMethod_ missing?

Reply:
Good idea but I have not seen such in any of Spryker modules. Not sure it is not prohibited